packages/config/src/index.ts

```diff
@@ -35,26 +36,6 @@ export type ConfigProcessOptions = {
   importFn?: (moduleId: string) => Promise<any>;
 };
 
-export async function parseConfig(
```

Comment:
Let's bring this back because this will be breaking change if someone is using it. We can put a reminder comment to remove it later because I know it is useless :/ .

Reply:
Fair enough, I saw it was unused internally and took the chance to do some cleanup.
I don't think it's documented so didn't think it could cause issues to people.
Will re-introduce.

Reply:
@santino it's an export in the TypeScript types, so documented or not, it's part of the API. @ardatan is correct in that it would be a breaking change.

Reply:
Yes, it makes sense and I have addressed it.
No breaking change introduced in the end just extended new functionalities to this exported function as well.